web/themes/custom/imaginelex/css/styles.css

```diff
@@ -1,3 +1,63 @@
 div#block-imaginelex-branding {
   max-width: 300px;
+}
```

Comment:
We should really set up SCSS and compiling of the files into one but I'm fine with this for now.